DEVNOTES.md

```diff
@@ -10,12 +10,17 @@ Clone the project, enter the project directory, and switch to the development br
 ~/FastqHeat$ git checkout dev
 ```
 
-Install dependencies with [`poetry`](https://python-poetry.org/):
+Install [`poetry`](https://python-poetry.org/), then install the project:
 
 ```bash
 ~/FastqHeat$ poetry install  # NOTE: includes dev dependencies
 ```
 
+> **NOTE**: to run commands within the project's virtual environment you will have
```

Comment:
it's needed to change this instruction to a clear one, point all needed commands which user has to do to get the result. Point by point, step by step